app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -1,5 +1,260 @@
-import React from 'react';
+import {
+  Area,
+  Button,
+  ButtonVariant,
+  Col,
+  FormikCheckbox,
+  FormikDropdown,
+  FormikText,
+  IOptionItem,
+  OptionItem,
+  Row,
+  Tab,
+  TabContainer,
+} from 'components';
+import { Formik } from 'formik';
+import { IContentApi, useApiEditor } from 'hooks';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useLookup } from 'store';
 
+import { PropertiesContentForm } from '.';
+import { formContentToApiContent } from './contentFormUtils';
+import { TranscriptContentForm } from './TranscriptContentForm';
+
+/**
+ * Content Form edit and create form for default view. Path will be appeneded with content id.
+ * @returns Edit/Create Form for Content
+ */
 export const ContentForm: React.FC = () => {
-  return <div>Snippet Form</div>;
+  const [active, setActive] = useState('properties');
+  const navigate = useNavigate();
+  const {
+    storeMediaTypes,
+    state: { mediaTypes },
+  } = useLookup();
+  const [mediaTypeOptions, setMediatTypeOptions] = useState<IOptionItem[]>([]);
+  const api = useApiEditor({ baseURL: `/api/editor/contents/` });
+  const lookupApi = useApiEditor({ baseURL: '/api' });
+  const path = window.location.pathname.toString();
+  const getContentId = () => {
+    return Number(path.substring(path.lastIndexOf('/') + 1));
+  };
+
+  const [mode] = useState<'edit' | 'create'>(getContentId() === 0 ? 'create' : 'edit');
+
+  const defaultValues: IContentApi = {
+    id: 0,
+    status: 0,
+    mediaTypeId: 0,
+    headline: '',
+    summary: '',
+    contentTypeId: 0,
+    source: '',
+    licenseId: 0,
+    page: '',
+    section: '',
+    transcription: '',
+    ownerId: 0,
+    createdOn: undefined,
+    seriesId: undefined,
+  };
+  const id = getContentId();
+  const [content, setContent] = useState<IContentApi>(defaultValues);
+  const [toggleCommentary, setToggleCommentary] = useState(true);
+
+  // include id when it is an update, no idea necessary when new content
+  const submitContent = async (data: IContentApi, contentId?: number) => {
+    mode === 'create' ? await api.addContent(data) : await api.updateContent(data, id);
+    navigate('/contents');
+  };
+
+  useEffect(() => {
+    api.findContent(id).then((data) => setContent(data));
+  }, [api, id]);
+
+  // Populate lookup values on initial render
+  useEffect(() => {
```

Comment:
Don't need to do this, but we should move logic like this to the `useLookup()` hook.